itest/tests/ingest.test.js

```diff
@@ -28,7 +28,12 @@ describe("Ingest tests", () => {
 
   const searchZql =
     "_path=conn proto=tcp | cut ts, id.orig_h, id.orig_p, id.resp_h, id.resp_p, proto | sort ts"
-  const sampleFiles = ["sample.pcap", "sample.tsv"]
+  const sampleFiles = [
+    "sample.pcap",
+    "sample.pcapng",
+    "sample.tsv",
+    "sample.ndjson"
+  ]
```

Comment:
This is awesome, and makes it less verbose. If one of these fails and I want to run it by itself. I don't think I'd be able to without changing code though right? I'd need to come in here and change the array to only have the item I'm looking for.

Reply:
Not a big deal though. I think it's worth it.

Reply:
@jameskerr the key is to make the test name contain the unique thing being tested. You can then use jest like so:
```
$ npm run itest -- -t "sample.pcapng" itest/tests/ingest.test.js

> Brim@0.9.1 itest /Users/mikeb/git/brim
> jest --config=itest/config.json --runInBand --verbose "-t" "sample.pcapng" "itest/tests/ingest.test.js"

[2020-05-21 10:16:15 info]: Running test: ingest of sample.pcapng
 PASS  itest/tests/ingest.test.js (10.2s)
  Ingest tests
    ✓ ingest of sample.pcapng (5928ms)
    ○ skipped ingest of sample.pcap
    ○ skipped ingest of sample.tsv
    ○ skipped ingest of sample.ndjson

Test Suites: 1 passed, 1 total
Tests:       3 skipped, 1 passed, 4 total
Snapshots:   1 passed, 1 total
Time:        10.537s, estimated 29s
Ran all test suites matching /itest\/tests\/ingest.test.js/i with tests matching "sample.pcapng".
```

Treat `-t` like a regex. `-t sample.pcap` matches two tests, so another way to get just `sample.pcap` would be:
```
$ npm run itest -- -t "sample.pcap$" itest/tests/ingest.test.js

> Brim@0.9.1 itest /Users/mikeb/git/brim
> jest --config=itest/config.json --runInBand --verbose "-t" "sample.pcap$" "itest/tests/ingest.test.js"

[2020-05-21 10:19:42 info]: Running test: ingest of sample.pcap
 PASS  itest/tests/ingest.test.js (9.678s)
  Ingest tests
    ✓ ingest of sample.pcap (5970ms)
    ○ skipped ingest of sample.pcapng
    ○ skipped ingest of sample.tsv
    ○ skipped ingest of sample.ndjson

Test Suites: 1 passed, 1 total
Tests:       3 skipped, 1 passed, 4 total
Snapshots:   1 passed, 1 total
Time:        10.015s, estimated 11s
Ran all test suites matching /itest\/tests\/ingest.test.js/i with tests matching "sample.pcap$".
```

Reply:
Oh wow, that's really cool. Makes sense now that I think about it.